tests/api_tests/cuda/test_cuda_add_documents.py

```diff
@@ -88,24 +88,6 @@ def test_add_documents_with_ids_twice(self):
         self.client.index(self.index_name_1).add_documents([d2], device="cuda", non_tensor_fields=[])
         assert d2 == self.client.index(self.index_name_1).get_document("56")
 
-    def test_add_batched_documents(self):
```

Comment:
is this worth deleting? What about adding the `client_batch_size` arg to the add_docs call?

Reply:
I thought that adding `client_batch_size` would be better suited as a client test, but I can do it here as well.